pkg/config/msp_test.go

```diff
@@ -11,13 +11,350 @@ import (
 	"crypto/ecdsa"
 	"crypto/x509"
 	"crypto/x509/pkix"
+	"encoding/pem"
 	"testing"
 
+	"github.com/golang/protobuf/proto"
+	cb "github.com/hyperledger/fabric-protos-go/common"
 	mb "github.com/hyperledger/fabric-protos-go/msp"
 	"github.com/hyperledger/fabric/common/tools/protolator"
+
 	. "github.com/onsi/gomega"
 )
 
+const (
```

Comment:
And how was this "arbitrary" certificate generated?

Reply:
I just found a couple of example certs from the x509 golang tests